<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bubble chart data generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/root/bubble.png</normaloff>:/root/bubble.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GO</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>63</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Files</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInputFile">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>25</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseInputFile">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>24</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>531</width>
     <height>521</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBgColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>276</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTextColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>316</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Border color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorderColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>356</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>34</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chart title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtChartTitle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>391</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Legend color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLegendColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>396</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGiamSanColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>276</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Giam San color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Giam Gia color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGiamGiaColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>316</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDungGiaColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>356</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dung Gia color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tang Gia color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTangGiaColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>396</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tang Tran color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTangTranColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>436</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chart rect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtChartWidth">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>116</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtChartHeight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>116</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(width x height)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(width x height)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtChartBgWidth">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>156</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chart size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtChartBgHeight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>156</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGridColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>436</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChartBgColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>476</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chart area color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>202</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y axis step</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbYAxisStep">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>196</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkYStepAutoCalculated">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>201</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto calculated</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Animation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAnimation">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>239</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Transition effect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chart sub title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtChartSubTitle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>72</y>
      <width>391</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtXMaxValue">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>116</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X axis max value</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>630</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
